### ISSUE TITLE
Attribution

### ISSUE THREAD

**Issue by perliedman** [mapbox/osm-bright#66]
Hi, this is more of a support request than an actual issue. If there is a more appropriate forum for this, please direct me there.

We have forked OSM Bright and made significant changes to the style, to adapt it to another data set than OSM (the Swedish land survey's data).

We're trying to understand what we have to do with regards to attribution. The license talks about source and binary form, but it is not entirely clear to me if rendered tiles are to be considered as "binary form".

In summary: what is the appropriate way to follow the license regarding rendered tiles? Should we reference Mapbox, Development Seed or this repository, or is this only required when redistributing the source (CartoCSS)?

On a side note, we want to redistribute the changes we've made, but have some more work to do regarding the required data mangling for the code to be usable to anyone else.
